<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Current Sgtk Environment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="0">
    <widget class="QLabel" name="label_comment">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Off Screen</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="context">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Your Current Context: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="3" colspan="2">
    <widget class="QTextEdit" name="textEdit_comment"/>
   </item>
   <item row="4" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>127</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="createPlayblast">
     <property name="text">
      <string>Playblast!</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="logo_example">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/sg_logo.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="sourceRes_cb">
        <item>
         <property name="text">
          <string>From render settings</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>From window</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="finalRes_label">
        <property name="text">
         <string>Final Resolution</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sourceRes_label">
        <property name="text">
         <string>Source Resolution</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_finalRes_h"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_scale">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_finalRes_w"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_framePadding">
        <property name="text">
         <string>FramePadding</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_framePadding">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_scale">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_focalLength"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5_cameraType">
        <property name="text">
         <string>Camera Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_cameraType"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4_passType">
        <property name="text">
         <string>Pass Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_passType">
        <item>
         <property name="text">
          <string>Greyshade</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Godcam</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stabilization</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wireframe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6_focalLength">
        <property name="text">
         <string>Focal length</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_1_frameRange">
        <property name="text">
         <string>Frame Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="frameRange_start">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="frameRange_end">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_format">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_format">
        <item>
         <property name="text">
          <string>avi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>image</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_encoding">
        <property name="text">
         <string>Encoding</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_encoding">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jpg</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>logo_example</zorder>
  <zorder>context</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_comment</zorder>
  <zorder>textEdit_comment</zorder>
  <zorder>checkBox</zorder>
  <zorder>createPlayblast</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
